sites/main-site/src/content/blog/2024/tools-3_0_0.md

```diff
@@ -0,0 +1,252 @@
+---
+title: 'nf-core/tools - 3.0.0'
+subtitle: 'What flavour of the nf-core template do you prefer?'
+headerImage: 'https://images.unsplash.com/photo-1583258292688-d0213dc5a3a8'
+headerImageAlt: 'Yellow and red apples on black plastic crate'
+pubDate: 2024-10-03T12:00:00+01:00
+authors:
+  - 'mirpedrol'
+  - 'mashehu'
+label:
+  - 'tools'
+---
+
```

Comment:
Need an intro paragraph. Why this is a big release. Why it's major semver. Why it's needed. Why now..